### PR TITLE
view all organisations link

### PR DESCRIPTION
**Pull request by RNR1**
improving #195

- added link to all organisations after the category photos.
- npm update.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,49 +11,49 @@
...
-    "eslint-config-airbnb": "^18.0.1",
-    "eslint-config-prettier": "^6.10.0",
-    "eslint-plugin-import": "^2.20.1",
+    "eslint-config-airbnb": "^18.1.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.2",
-    "eslint-plugin-react": "^7.18.3",
+    "eslint-plugin-react": "^7.19.0",
```
Comment (jasonLaster) [inline package.json:56]: could you move these package changes into their own PR?

File: src/pages/index.js [hunk 2]
```diff
@@ -27,6 +27,11 @@ export default function IndexPage({ data }) {
             <TopicCard category={cat} key={cat.name} />
           ))}
         </div>
+        <div className="text-center">
+          <Link to="/organizations" className="text-blue-400">
+            View All Organizations &gt;
```
Comment (jasonLaster) [inline src/pages/index.js:32]: Lets use the chevron key
https://www.compart.com/en/unicode/U+203A

File: src/pages/index.js [hunk 3]
```diff
@@ -27,6 +27,11 @@ export default function IndexPage({ data }) {
             <TopicCard category={cat} key={cat.name} />
           ))}
         </div>
+        <div className="text-center">
+          <Link to="/organizations" className="text-blue-400">
```
Comment (jasonLaster) [inline src/pages/index.js:31]: I think the designs were a darker color
https://www.figma.com/file/GvYYLhGvEsLxS1T677r0Ox/Climatescape?node-id=504%3A1867

We dont have to be pixel-perfect, but something darker would be nice

File: package.json (hunk 1 above)
Comment (RNR1) [reply package.json]: done, made a reset to both package and package-lock

File: src/pages/index.js (hunk 2 above)
Comment (RNR1) [reply src/pages/index.js]: done, i added it inside a span because it was smaller compared to the text

File: src/pages/index.js (hunk 3 above)
Comment (RNR1) [reply src/pages/index.js]: updated as well, tailwind didn't had that exact dark blue but i changed it to closest one, the other option is to pick it separately with inline styles.

**Comment by jasonLaster** [comment]
Looks good to me
